src/main/java/uk/gov/hmcts/reform/sscs/exception/GetHearingException.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.sscs.exception;
+
+public class GetHearingException extends Exception {
```

Comment:
Could we have a parent exception for these, then pass these as the cause in ProcessMessageService?